.github/workflows/BackendCI.yaml

```
@@ -0,0 +1,44 @@
+name: backend-lint
+on:
+  push:
+    branches:
+      - "**"
+    paths:
+      - "backend/**"
+
```

```suggestion
      - ".github/workflows/BackendCI.yaml"
```
so it triggers on changes to self